### ISSUE TITLE
[TF2] Matchmaking Settings Party on Same team

### ISSUE THREAD

**Issue by kay0115** [ValveSoftware/Source-1-Games#4349]
For a long time in the matchmaking settings there has been a "Coming Soon" on no longer keeping party members on the same team. It would be nice if this was finally added

**Comment by condor00fr** [comment]
Yeah, it would be good to see this, having you and your friends being able to split into two teams and join a match would be freaking unbelievable! Only issue is that bots could abuse this to overrun the games. But even then they do this already so.... add this feature pls.

**Comment by jh34ghu43gu** [comment]
For reference https://imgur.com/kfnhaR2

Also I always thought this meant if it was implemented you couldn't be auto balanced off your party's team?
